### ISSUE TITLE
any other 3d models can be used in the demo?

### ISSUE THREAD

**Issue by peterchen2001** [yukihiko/ThreeDPoseUnitySample#13]
Great job, I am trying to replace or recreate other 3d models referring to the Unity-chan in the demo, but the animation doesn't work as Unity-chan. Any other 3d models to download or any guidline to create such 3d models that can be used in this demo?

**Comment by yukihiko** [comment]
Thank you.
1. Add a GameObject at the character's nose position.
2. Add AvatarController.cs to the character and set the Nose, onnx file, and low pass filter.
3.Set in ThreeDPoseScript of Main Camera
4. Some characters do not have bones. Many have no thumb or middle finger. Please comment out the relevant part of AvatarController.

How about this?

**Comment by peterchen2001** [comment]
Yes, that works. However, after using my own avatar(has same bones like Unity-chan), i see some compoents become twisty, such as the loin and the knees. Any ideas?

See the video what i got: http://www.magic3d.top:7012/Data/Video/3260a64f2e857b4a1e193256e0109fc5.mp4

**Comment by yukihiko** [comment]
Does it work with normal motion data?

**Comment by maithstartup** [comment]
> Thank you.
> 
> 1. Add a GameObject at the character's nose position.
> 2. Add AvatarController.cs to the character and set the Nose, onnx file, and low pass filter.
>    3.Set in ThreeDPoseScript of Main Camera
> 3. Some characters do not have bones. Many have no thumb or middle finger. Please comment out the relevant part of AvatarController.
> 
> How about this?

**Comment by maithstartup** [comment]
> Thank you.
> 
> 1. Add a GameObject at the character's nose position.
> 2. Add AvatarController.cs to the character and set the Nose, onnx file, and low pass filter.
>    3.Set in ThreeDPoseScript of Main Camera
> 3. Some characters do not have bones. Many have no thumb or middle finger. Please comment out the relevant part of AvatarController.
> 
> How about this?

Can you please elaborate on this, I have the project working for unity chan and I have imported another avatar with nearly similar bone structure , should do the above mention steps in the same scene or create a new scene, please guide me

**Comment by arunvaswani1** [comment]
> Thank you.
> 
> 1. Add a GameObject at the character's nose position.
> 2. Add AvatarController.cs to the character and set the Nose, onnx file, and low pass filter.
>    3.Set in ThreeDPoseScript of Main Camera
> 3. Some characters do not have bones. Many have no thumb or middle finger. Please comment out the relevant part of AvatarController.
> 
> How about this?

tried this, it didn't work. i get this error ====
NullReferenceException: Object reference not set to an instance of an object
ThreeDPoseScript+<PoseUpdate>d__40.MoveNext ()

Please guide
